applications/DEM_application/custom_conditions/RigidFace.h

```diff
@@ -47,14 +43,43 @@ class RigidFace3D : public DEMWall
     // Destructor
     virtual ~RigidFace3D();
 
+    class ParticleDataBuffer
```

Comment:
Yes, I will change it in the next request, which is the one that will be effective. This one will be erased